shared/django_apps/rollouts/migrations/0001_initial.py

```diff
@@ -0,0 +1,127 @@
+    # DROP TABLE "feature_flag_variants";
+    # ALTER TABLE "new__feature_flag_variants" RENAME TO "feature_flag_variants";
+    # CREATE INDEX "feature_flag_variants_feature_flag_id_fa3a4c02" ON "feature_flag_variants" ("feature_flag_id");
+    # CREATE INDEX "feature_fla_feature_15a078_idx" ON "feature_flag_variants" ("feature_flag_id");
+    # COMMIT;
+
+    operations = [
+        migrations.CreateModel(
+            name="FeatureFlag",
+            fields=[
+                (
+                    "name",
+                    models.CharField(max_length=200, primary_key=True, serialize=False),
+                ),
+                (
+                    "proportion",
+                    models.DecimalField(decimal_places=3, default=0, max_digits=4),
+                ),
+                (
+                    "salt",
+                    models.CharField(
+                        default=shared.django_apps.rollouts.models.default_random_salt,
+                        max_length=10000,
+                    ),
+                ),
+            ],
+            options={
+                "db_table": "feature_flags",
+            },
+        ),
+        migrations.CreateModel(
+            name="FeatureFlagVariant",
+            fields=[
+                (
+                    "name",
+                    models.CharField(max_length=200, primary_key=True, serialize=False),
+                ),
+                (
+                    "proportion",
+                    models.DecimalField(decimal_places=3, default=0, max_digits=4),
+                ),
+                ("enabled", models.BooleanField(default=False)),
+                (
+                    "override_owner_ids",
+                    django.contrib.postgres.fields.ArrayField(
+                        base_field=models.IntegerField(), default=list, size=None
+                    ),
+                ),
+                (
+                    "override_repo_ids",
+                    django.contrib.postgres.fields.ArrayField(
+                        base_field=models.IntegerField(), default=list, size=None
+                    ),
+                ),
+                (
+                    "feature_flag",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        related_name="variants",
+                        to="rollouts.featureflag",
+                    ),
+                ),
+            ],
+            options={
+                "db_table": "feature_flag_variants",
+            },
+        ),
+        migrations.AddConstraint(
+            model_name="featureflag",
+            constraint=models.UniqueConstraint(
+                fields=("name",), name="feature_flag_name"
+            ),
+        ),
```

Comment:
this column is designated the primary key, right? doesn't that already require uniqueness?